pp_netlib/network.py

```diff
@@ -39,82 +41,137 @@ def __init__(self, ref_list, query_list, assignments, outdir, model_type, use_gp
+        #         for (src, dest, weight) in zip(extra_sources, extra_targets, extra_weights):
+        #                 prev_edges.append((src, dest, weight))
+
+        #     else:
+        #         extra_sources, extra_targets = process_previous_network(previous_network = previous_network, adding_qq_dists = adding_qq_dists, old_ids = old_ids, previous_pkl = previous_pkl, vertex_labels = vertex_labels, weights = (weights is not None), use_gpu = use_gpu)
+        #         for (src, dest, weight) in zip(extra_sources, extra_targets):
+        #                 prev_edges.append((src, dest))
+
+
+        #     self.graph.add_edge_list(prev_edges) ## add previous edge list to newly made graph
+
+        #     self.edges = [edge for edge in self.graph.edges()]
+
+
+        ################################
+        #   TODO TODO TODO TODO TODO   #
+        #   TODO TODO TODO TODO TODO   #
+        ################################
+        # else:
+        #     # benchmarking concurs with https://stackoverflow.com/questions/55922162/recommended-cudf-dataframe-construction
+        #     if len(network_data_df) > 1:
+        #         edge_array = cp.array(network_data_df, dtype = np.int32)
+        #         edge_gpu_matrix = cuda.to_device(edge_array)
+        #         gpu_graph_df = cudf.DataFrame(edge_gpu_matrix, columns = ["source","destination"])
+        #     else:
+        #         # Cannot generate an array when one edge
+        #         gpu_graph_df = cudf.DataFrame(columns = ["source","destination"])
+        #         gpu_graph_df["source"] = [network_data_df[0][0]]
+        #         gpu_graph_df["destination"] = [network_data_df[0][1]]
+
+        #     if weights is not None:
+        #         gpu_graph_df["weights"] = weights
+
+        #     if previous_network is not None:
+        #         G_extra_df = cudf.DataFrame()
+        #         G_extra_df["source"] = extra_sources
+        #         G_extra_df["destination"] = extra_targets
+        #         if extra_weights is not None:
+        #             G_extra_df["weights"] = extra_weights
+        #         gpu_graph_df = cudf.concat([gpu_graph_df,G_extra_df], ignore_index = True)
+
+        #     # direct conversion
+        #     # ensure the highest-integer node is included in the edge list
+        #     # by adding a self-loop if necessary; see https://github.com/rapidsai/cugraph/issues/1206
+        #     max_in_vertex_labels = len(vertex_labels)-1
+        #     use_weights = False
+        #     if weights:
+        #         use_weights = True
+        #     self.graph = add_self_loop(gpu_graph_df, max_in_vertex_labels, weights = use_weights, renumber = False)
                 
-        return self.graph
+        # return self.graph
```

Comment:
Remove all of this, or store it in a tmp file somewhere for now. Best to keep your main functions clean and easy to read while developing

Reply:
done